python/graphstorm/model/utils.py

```diff
@@ -738,7 +736,74 @@ def save_shuffled_node_embeddings(shuffled_embs, save_embed_path, save_embed_for
 
     if rank == 0:
         with open(os.path.join(save_embed_path, "emb_info.json"), 'w', encoding='utf-8') as f:
-            f.write(json.dumps(emb_info))
+            json.dump(emb_info, f, indent=4)
+
+def save_full_node_embeddings(g, save_embed_path,
+                              embeddings,
+                              node_id_mapping_file,
+                              save_embed_format="pytorch"):
+    """ Save full node embeddings.
```

Comment:
can you explain the difference of this function and `save_embeddings`?

Reply:
In `save_full_node_embeddings`, node embeddings saved by this function has corresponding nid saved.
While `save_embeddings` can be deprecated. But as it is used by bert caching system, we should not remove it now.

Reply:
Update the docstring